web/themes/custom/votegov/templates/paragraph/paragraph--registration-tool.html.twig

```diff
@@ -30,23 +30,22 @@
     </div>
     <form id="vote-registration-tool__form" class="vote-registration-tool__form">
       <label class="{{ not content.field_body | render ? 'usa-sr-only' }} vote-registration-tool__label"
-             for="vote-registration-tool__input">
+            for="vote-registration-tool__input">
         {{ content.field_form_heading | field_value }}
       </label>
       <input type="text" class="vote-registration-tool__input usa-input maxw-none"
-             id="vote-registration-tool__input" autocomplete="off"
-             placeholder="{{ content.field_placeholder | field_value }}" data-test="stateInput"
-             aria-autocomplete="list" aria-expanded="true" aria-controls="vote-registration-tool__results">
+            id="vote-registration-tool__input" autocomplete="off"
+            placeholder="{{ content.field_placeholder | field_value }}" 
+            aria-autocomplete="list" aria-controls="vote-registration-tool__results" data-test="stateInput">
```

Comment:
i think the `aria-autocomplete` is not needed... from what i am reading it would be needed if the input filed autocompletes -> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-autocomplete

Reply:
I agree. I think we could remove it. This field is not an autocomplete field but rather a filter input for the list below.

Reply:
@clmedders I think we can keep this, because the suggestions are made in a list below the input.
"It distinguishes between two models: the inline model (aria-autocomplete="inline"), which presents a single predicted value and the list model (**aria-autocomplete="list"**), which presents a collection of possible values in a separate element that pops up next to or below the text input"

Reply:
@rayestrada I see your comment now..I think I get it.